modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAbstractIntegrationTest.java

```diff
@@ -193,6 +199,12 @@ public void createApplication(OIDCApplication application) throws Exception {
                     .toArray(new InboundAuthenticationRequestConfig[authRequestList.size()]));
         }
 
+        appMgtclient.updateApplicationData(serviceProvider);
```

Comment:
Shall we use the newly introduced method `updateApplication` ?
```suggestion
       updateApplication(serviceProvider);
```

Reply:
Fixed https://github.com/wso2/product-is/pull/11407/commits/4fe18ee0052418875fffa6a22be116143ea73ebb